docs/project-types/magento2.md

```diff
@@ -1,2 +0,0 @@
-- `magento2-project`: [Youwe Coding Standard Magento2](../../config/magento2)
```

Comment:
suggest: either remove the referenced files too or reference them in the new README table (assuming no otherreferences exist anymore)